Tests/SwiftLintFrameworkTests/NumberSeparatorRuleTests.swift

```diff
@@ -58,4 +58,41 @@ class NumberSeparatorRuleTests: XCTestCase {
 
         verifyRule(description, ruleConfiguration: ["minimum_fraction_length": 5])
     }
+
+    func testNumberSeparatorWithValidRanges() {
+        let nonTriggeringExamples = [
+            "let foo = 1950",
+            "let foo = 1985",
+            "let foo = 2020",
+            "let foo = 2.10042",
+            "let foo = 2.833333"
+        ]
+        let triggeringExamples = [
+            "let foo = ↓1000",
+            "let foo = ↓2100",
```

Comment:
it'd be good to add an example here with a number in the valid range formatted with `_`

Reply:
Fixed.